Reference/Management/Models/index.md

```diff
@@ -5,6 +5,12 @@ versionTo: 9.0.0
 
 # Persistence Models reference
 
+:::note
+As of the release of Umbraco 10 we will no longer be updating the articles in this section.
```

Comment:
```suggestion
As of Umbraco 10 release, we will no longer be updating the articles in this section.
```

Reply:
Perhaps instead we could do: 
Since the release of Umbraco 10, we will no longer ... 

Would that be better?

Reply:
Yes that sounds better 🙌